components/layout_2020/flexbox/layout.rs

```diff
@@ -1874,24 +1875,34 @@ impl FlexItem<'_> {
         line_cross_size: Au,
         propagated_baseline: Au,
         max_propagated_baseline: Au,
+        wrap_reverse: bool,
     ) -> Au {
+        let ending_alignment = line_cross_size - (*used_cross_size + self.pbm_auto_is_zero.cross);
```

Comment:
The parenthesis could be removed, but fine either way

```suggestion
        let ending_alignment = line_cross_size - *used_cross_size - self.pbm_auto_is_zero.cross;
```